distributed/http/scheduler/prometheus.py

```diff
@@ -2,6 +2,9 @@
 
 from ..utils import RequestHandler
 from ...scheduler import ALL_TASK_STATES
+from ..semaphore.prometheus import SemaphoreMetricExtension
+
+DEFAULT_METRIC_EXTENSIONS = (SemaphoreMetricExtension,)
```

Comment:
@mrocklin Opinions on this?

Reply:
This could work.  Maybe we can make this slightly more generic and have something like ...


```python
from ..semaphore.prometheus import SemaphoreMetricExtension


class _PrometheusCollector:
    ...

collectors = [
    _PrometheusCollector, 
    SemaphoreMetricCollector,
]

...
```

This way everything is on the same baseline?